src/Symfony/Component/AssetMapper/AssetMapperDevServerSubscriber.php

```diff
@@ -134,17 +139,28 @@ public function onKernelRequest(RequestEvent $event): void
             ->setPublic()
             ->setMaxAge(604800)
             ->setImmutable()
-            ->setEtag($asset->digest)
-        ;
+            ->setEtag($asset->digest);
+
+        $response->headers->set('X-DevMappedAsset', true);
```

Comment:
To be consistent/more readable, a dash should probably separate words. What about something simpler as well. Maybe `X-Assets-Dev`?

Reply:
👍